src/main/java/io/cryostat/net/web/http/api/v2/graph/ArchivedRecordingsFetcher.java

```diff
@@ -113,5 +128,6 @@ static class Archived {
 
     static class AggregateInfo {
         Long count;
+        Long size;
```

Comment:
Not sure why I didn't catch this when the `count` was added, but can these two fields be changed to the primitive `long` instead?

Reply:
done!